web/src/features/charts/elements/AreaGraphLayers.tsx

```diff
@@ -75,6 +75,7 @@ function AreaGraphLayers({
         // Therefore, we copy all datapoints and make sure
         // both a start and an end are present to ensure
         // proper display of missing points
+        // eslint-disable-next-line unicorn/no-useless-spread
         const datapoints = [
```

Comment:
if you let eslint make changes it wants here, then you'll get warning when using datapoints below. i confess i don't fully understand it so i left it alone.

Reply:
It is likley cause we are calling the flat function on the entire array which is making the spread redundant, but please do not add new eslint-disable comments.

Reply:
ok do you prefer this?